### PR TITLE
USERGRID-933: org config refactor

### PR DESCRIPTION
**Pull request by mdunker**
Add 4 more config items, and keep workflow organization UUID in the token for providing the correct emails and email templates during the confirmation/activation process.

### CODE REVIEW

File: stack/services/src/main/java/org/apache/usergrid/management/cassandra/ManagementServiceImpl.java [hunk 1]
```diff
@@ -3184,20 +3261,21 @@ private boolean getBooleanProperty(String key) {
...
+        return new OrganizationConfig(orgConfigProperties);
+    }
+
 
     @Override
     public OrganizationConfig getOrganizationConfigByName( String organizationName ) throws Exception {
 
-        if ( organizationName == null ) {
-            return null;
+        if ( organizationName == null || organizationName.equals("management")) {
```
Comment (mdunker) [inline stack/services/src/main/java/org/apache/usergrid/management/cassandra/ManagementServiceImpl.java]: shouldn't be hardcoded

File: stack/services/src/main/java/org/apache/usergrid/management/cassandra/OrganizationConfigPropsImpl.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+import java.util.*;
+
+import static org.apache.commons.lang.StringUtils.isBlank;
+
+
+public class OrganizationConfigPropsImpl implements OrganizationConfigProps {
+    private static final Logger logger = LoggerFactory.getLogger( OrganizationConfigPropsImpl.class );
+
+    private static final String DEFAULT_CONNECTION_PARAM_DEFAULTVALUE = "all";
```
Comment (mdunker) [inline stack/services/src/main/java/org/apache/usergrid/management/cassandra/OrganizationConfigPropsImpl.java]: from config file?

Comment (mdunker) [inline stack/services/src/main/java/org/apache/usergrid/management/cassandra/OrganizationConfigPropsImpl.java]: pull from default system config file?

File: stack/services/src/main/java/org/apache/usergrid/management/cassandra/OrganizationConfigPropsImpl.java [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+    private static final String ADMIN_SYSADMIN_EMAIL_DEFAULTVALUE = null; // null will fall back to system level admin
+    private static final String ADMIN_ACTIVATION_URL_DEFAULTVALUE = ""; // should be configured in properties file
+    private static final String ADMIN_CONFIRMATION_URL_DEFAULTVALUE = ""; // should be configured in properties file
+    private static final String ADMIN_RESETPW_URL_DEFAULTVALUE = ""; // should be configured in properties file
+
+    private static final Map<String, String> noConfigDefaults = new HashMap<>();
+    static {
+        noConfigDefaults.put(PROPERTIES_DEFAULT_CONNECTION_PARAM, DEFAULT_CONNECTION_PARAM_DEFAULTVALUE);
```
Comment (mdunker) [inline stack/services/src/main/java/org/apache/usergrid/management/cassandra/OrganizationConfigPropsImpl.java]: move DEFAULTVALUE to front of name

File: stack/services/src/main/java/org/apache/usergrid/security/tokens/cassandra/TokenServiceImpl.java [hunk 4]
```diff
@@ -267,6 +288,7 @@ public void importToken(String token, TokenCategory tokenCategory, String type,
             Assert.notNull( principal.getType() );
             Assert.notNull( principal.getApplicationId() );
             Assert.notNull( principal.getUuid() );
+            // workflowOrgId would be taken from state
```
Comment (mdunker) [inline stack/services/src/main/java/org/apache/usergrid/security/tokens/cassandra/TokenServiceImpl.java]: delete line

**Comment by michaelarusso** [comment]
:+1:

**Pull request merged**